string button_mode # Hold, Toggle
bool buzzer_active # true, false
